### ISSUE TITLE
if the stock data is pure random, can i get a loss near zero in train?

### ISSUE THREAD

**Issue by war3gu** [AI4Finance-Foundation/FinRL#479]
i think i can, but i will get a bad result in test part.

**Comment by Burntt** [comment]
If you train long enough on the same train dataset, you will be overfitting that train set completely. That means your model will be able to predict the random stock data exactly, being able to reduce certain loss functions to a complete minimum.

However, as you point out, this delivers terrible performance on your test set. I recommend this video:

https://www.youtube.com/watch?v=Anq4PgdASsc

**Comment by war3gu** [comment]
> If you train long enough on the same train dataset, you will be overfitting that train set completely. That means your model will be able to predict the random stock data exactly, being able to reduce certain loss functions to a near minimum. It does not matter if your data is random or not.
> 
> However, as you point out, this delivers terrible performance on your test set. I recommend this video:
> 
> https://www.youtube.com/watch?v=Anq4PgdASsc

thank you for your advise
